Mathlib/Computability/Primrec.lean

```diff
@@ -1113,6 +1113,19 @@ theorem list_join : Primrec (@List.join α) :=
     dsimp; induction l <;> simp [*]
 #align primrec.list_join Primrec.list_join
 
+theorem list_bind {f : α → List β} {g : α → β → List σ} (hf : Primrec f) (hg : Primrec₂ g) :
+    Primrec (fun a => (f a).bind (g a)) := list_join.comp (list_map hf hg)
+
+theorem option_toList : Primrec (Option.toList : Option α → List α) :=
+  (option_casesOn Primrec.id (const [])
+    ((list_cons.comp Primrec.id (const [])).comp₂ Primrec₂.right)).of_eq
+  (fun o => by rcases o <;> simp)
+
+theorem list_filterMap {f : α → List β} {g : α → β → Option σ}
```

Comment:
```suggestion
theorem listFilterMap {f : α → List β} {g : α → β → Option σ}
```
or just
```suggestion
theorem filterMap {f : α → List β} {g : α → β → Option σ}
```
because I think `filterMap` unambiguously means `List.filterMap`